mantidproject/mantid
# Reduce Pylint warnings in ISIS_reduction_steps.py

**To Test:**
- Code review
- All tests should pass
- See that PyLint build reports less warnings for isis_reduction_steps.py than previously

**Release Notes**
This is an internal maintenance issue with no impact on Mantid itself, so no update is needed for release notes.

Fixes #15107 
Fixes #14738
